src/main/resources/openapi/model/api.yaml

```
@@ -13,6 +13,46 @@ tags:
 #    description: "local host"
 
 paths:
+  /notification/sms/{refId}:
+    post:
+      description: Get a single message by reference id.
```

can the description be more descriptive please, its an sms notification message
```suggestion
      description: https://swagger.io/docs/specification/paths-and-operations/
```